### ISSUE TITLE
Make Failed with latest OpenCV

### ISSUE THREAD

**Issue by khairu-aqsara** [pjreddie/darknet#1897]
I, tried to compile with Latest OpenCV but getting an Error
```
./src/image_opencv.cpp:12:1: fatal error: unknown type name 'IplImage'
IplImage *image_to_ipl(image im)
```

Using Mac OSx Mojave, OpenCV installed via `brew`

```
darknet [master] ⚡  pkg-config --list-all | grep opencv
opencv4               OpenCV - Open Source Computer Vision Library

>>> import cv2
>>> cv2.__version__
'4.1.2'
>>>
```

OS: Mac OSx Mojava
Python : 3.7.4
OpenCV : 4.1.2

**Comment by beatgeek** [comment]
I can confirm same error on same version of Python and MacOS.

**Comment by beatgeek** [comment]
I posted this too hastily. I was able to resolve this issue with the addition of `--stdc++11` in the Makefile as well as make sure the libraries path is corrected to opencv4

**Comment by hnishi** [comment]
I confirmed same error too for opencv 4.1.2.
I added `-std=c++11` and corrected paths to opencv libraries in the Makefile but it did not work. 
FInally, I downgraded  opencv to version 3.
It resolved this issue.

my Makefile
https://github.com/hnishi/darknet/commit/37bb091c1e15f73866221f38aa9d09db61279f31

**Comment by Tanergol** [comment]
the fix is here : https://github.com/pjreddie/darknet/pull/1348/commits/24cff08086b573c3341e91d072430c9f624a2208

**Comment by yudhiesh1997** [comment]
> the fix is here : [24cff08](https://github.com/pjreddie/darknet/commit/24cff08086b573c3341e91d072430c9f624a2208)

Hey i did that now have this error 
opencv.cpp -o obj/image_opencv.o
./src/image_opencv.cpp:5:10: fatal error: 'opencv2/opencv.hpp' file not found
#include "opencv2/opencv.hpp"
         ^~~~~~~~~~~~~~~~~~~~

**Comment by lukasguldstoev** [comment]
https://www.py4u.net/discuss/1956607
this fixed it for me --std==c++1 ...
furthermore i had to move /usr/local/opencv4/opencv2 to /usr/local/opencv2  (basically just copying the directory inside opencv4 and move it out directly to local
and add some code since there seems to be a bug in darknet when using opencv 4.xx
